tool/frame.rkt

```diff
@@ -61,9 +61,11 @@
                               (send current-project set path)
                               (show-real-area)
                               (send c set-label "Hide the Project Viewer"))])))]
-               ;;; c+y open project viewer
+               ;;; c+y open project viewer (c+s+y when using Windows OS)
```

Comment:
c + y
c + s + y

Reply:
are you sure? adding space look ugly to me.

Reply:
Oh, I mean change line

Reply:
```
;; c+y on Linux, MacOS \n
;; c+s+y on Windows \n
```
something like that?